mining-buddy.lic

```diff
@@ -32,7 +32,17 @@ class MiningBuddy
   end
 
   def buy_deed_packet
-    order_item(8775, 14)
+    hometown = get_settings.hometown
```

Comment:
thats fine with me, pass the room number and order number as arguments to `buy_deed_packet`?